src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/DisputeService.java

```diff
@@ -66,6 +67,15 @@ public void save(Dispute dispute) {
 		for (DisputedCount disputedCount : dispute.getDisputedCounts()) {
 			disputedCount.setId(null);
 		}
+		if (dispute.getLegalRepresentation() != null) {
+			dispute.getLegalRepresentation().setId(null);
+		}
+		if (dispute.getViolationTicket() != null) {
+			dispute.getViolationTicket().setId(null);
+			for (ViolationTicketCount violationTicketCount : dispute.getViolationTicket().getViolationTicketCounts()) {
+				violationTicketCount.setId(null);
+			}
+		}
```

Comment:
Enhanced the OracleApi.  I noticed that the id used was an 0000-000... guid, not a generated one.  The id needs to be empty so hibernate can generate a new id for the new record.

Reply:
Guid is a value type and the default is Guid.Empty which is all zeros.

Reply:
Correct, but in the oracle-data-api, hibernate needs to have the id be null so it'll automatically generate a new value and INSERT the record. Otherwise, if a value already exists (even Guid.Empty all zeros), it'll use that and do an UPDATE instead.